code/obj/item/misc_weapons.dm

```diff
@@ -2207,3 +2211,34 @@ obj/item/whetstone
 	New()
 		..()
 		src.setItemSpecial(/datum/item_special/rangestab)
+
+
+
+
+/obj/item/gang_machete
+	name = "machete"
+	desc = "A hefty, unbalanced blade. Wielding it fills you with thoughts of savagery."
+	icon = 'icons/obj/items/weapons.dmi'
+	icon_state = "machete"
+	item_state = "welder_machete"
+	inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+	hit_type = DAMAGE_CUT
+	flags = FPRINT | USEDELAY
+	force = 25
+	click_delay = 16 DECI SECONDS //unbalanced blade
+	throwforce = 5
+	throw_speed = 1
+	throw_range = 5
+	contraband = 4
+	w_class = W_CLASS_NORMAL
+	tool_flags = TOOL_CUTTING
+	attack_verbs = "slashes"
+	hitsound = 'sound/impact_sounds/Blade_Small_Bloody.ogg'
+
+	New()
+		..()
+		src.AddComponent(/datum/component/bloodflick)
```

Comment:
Not sure if this an issue with the base component but this seems to cause a runtime on the third attack of each chain
![image](https://github.com/goonstation/goonstation/assets/20713227/3a843a4d-578a-488c-873f-69b5e71091b4)

Reply:
base class error - idk exactly waht the fix is atm.

Reply:
probably visible_message